Capri.Database/Entities/Configuration/SeedGetter.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Identity;
+using Newtonsoft.Json;
+using Capri.Database.Entities.Identity;
+
+namespace Capri.Database.Entities.Configuration
+{
+    public static class SeedGetter
+    {
+        public static Guid DeanRoleId { get; private set; } = Guid.NewGuid();
+        public static Guid PromoterRoleId { get; private set; } = Guid.NewGuid();
+        public static Guid StudentRoleId { get; private set; } = Guid.NewGuid();
+
+        public static List<User> Users { get; private set; } = new List<User>();
+        public static List<Student> Students { get; private set; } = new List<Student>();
+        public static List<Promoter> Promoters { get; private set; } = new List<Promoter>();
+        public static List<User> DeanEmployees { get; private set; } = new List<User>();
+        public static List<Institute> Institutes { get; private set; } = new List<Institute>();
+        public static List<Faculty> Faculties { get; private set; } = new List<Faculty>();
+        public static List<Course> Courses { get; private set; } = new List<Course>();
+        public static List<Proposal> Proposals { get; private set; } = new List<Proposal>();
+        private static DataSeed Seed = LoadDataSeed();
+
+        private static DataSeed LoadDataSeed()
+        {
+            var jsonString = File.ReadAllText("../Capri.Database/Entities/Configuration/dataseed.json");
```

Comment:
Use some async method for io operations. Also It will be better to use some path methods for getting current path.

Reply:
Done

Reply:
Done